authorization/http_server_test.go

```diff
@@ -144,65 +144,57 @@ func TestService_handlePostAuthorization(t *testing.T) {
 
-			s := itesting.NewTestInmemStore(t)
-			storage, err := NewStore(s)
-			if err != nil {
-				t.Fatal(err)
-			}
+				storage, err := NewStore(context.Background(), s, useHashedTokens)
+				require.NoError(t, err)
 
-			svc := NewService(storage, tt.fields.TenantService)
+				svc := NewService(storage, tt.fields.TenantService)
 
-			handler := NewHTTPAuthHandler(zaptest.NewLogger(t), svc, tt.fields.TenantService)
-			router := chi.NewRouter()
-			router.Mount(handler.Prefix(), handler)
+				handler := NewHTTPAuthHandler(zaptest.NewLogger(t), svc, tt.fields.TenantService)
+				router := chi.NewRouter()
+				router.Mount(handler.Prefix(), handler)
 
-			req, err := newPostAuthorizationRequest(tt.args.authorization)
-			if err != nil {
-				t.Fatalf("failed to create new authorization request: %v", err)
-			}
-			b, err := json.Marshal(req)
-			if err != nil {
-				t.Fatalf("failed to unmarshal authorization: %v", err)
-			}
+				req, err := newPostAuthorizationRequest(tt.args.authorization)
+				require.NoError(t, err)
 
-			r := httptest.NewRequest("GET", "http://any.url", bytes.NewReader(b))
-			r = r.WithContext(context.WithValue(
-				context.Background(),
-				httprouter.ParamsKey,
-				httprouter.Params{
-					{
-						Key:   "userID",
-						Value: fmt.Sprintf("%d", tt.args.session.UserID),
-					},
-				}))
+				b, err := json.Marshal(req)
+				require.NoError(t, err)
 
-			w := httptest.NewRecorder()
+				r := httptest.NewRequest("GET", "http://any.url", bytes.NewReader(b))
+				r = r.WithContext(context.WithValue(
+					context.Background(),
+					httprouter.ParamsKey,
+					httprouter.Params{
+						{
+							Key:   "userID",
+							Value: fmt.Sprintf("%d", tt.args.session.UserID),
+						},
+					}))
 
-			ctx := icontext.SetAuthorizer(context.Background(), tt.args.session)
-			r = r.WithContext(ctx)
+				w := httptest.NewRecorder()
 
-			handler.handlePostAuthorization(w, r)
+				ctx := icontext.SetAuthorizer(context.Background(), tt.args.session)
+				r = r.WithContext(ctx)
 
-			res := w.Result()
-			content := res.Header.Get("Content-Type")
-			body, _ := io.ReadAll(res.Body)
+				handler.handlePostAuthorization(w, r)
 
-			if res.StatusCode != tt.wants.statusCode {
-				t.Logf("headers: %v body: %s", res.Header, body)
-				t.Errorf("%q. handlePostAuthorization() = %v, want %v", tt.name, res.StatusCode, tt.wants.statusCode)
-			}
-			if tt.wants.contentType != "" && content != tt.wants.contentType {
-				t.Errorf("%q. handlePostAuthorization() = %v, want %v", tt.name, content, tt.wants.contentType)
-			}
-			if diff, err := jsonDiff(string(body), tt.wants.body); diff != "" {
-				t.Errorf("%q. handlePostAuthorization() = ***%s***", tt.name, diff)
-			} else if err != nil {
-				t.Errorf("%q, handlePostAuthorization() error: %v", tt.name, err)
-			}
-		})
+				res := w.Result()
+				contentType := res.Header.Get("Content-Type")
+				body, err := io.ReadAll(res.Body)
+				require.NoError(t, err)
```

Comment:
Similar to above - small string with context.

Reply:
No value added.